.github/workflows/wit.yaml

```diff
@@ -0,0 +1,59 @@
+name: Publish WIT package
+
+on:
+  push:
+    tags:
+    - 'wit-v*'
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    steps:
+    - uses: actions/checkout@eef61447b9ff4aafe5dcd4e0bbf5d482be7e7871
+    - name: Extract tag context
+      id: ctx
+      run: |
+          version=${GITHUB_REF_NAME#wit-v}
+          echo "version=${version}" >> "$GITHUB_OUTPUT"
+          echo "tarball=wit-${version}.tar.gz" >> "$GITHUB_OUTPUT"
+          echo "version is ${version}"
+    - name: Download wkg
+      shell: bash
+      run: |
+        curl --fail -L https://github.com/bytecodealliance/wasm-pkg-tools/releases/download/v0.7.4/wkg-x86_64-unknown-linux-gnu -o wkg
+        chmod +x wkg;
+        echo "$(realpath .)" >> "$GITHUB_PATH";
```

Comment:
Can we use one of the pre-built actions ([`cache-cargo-install-action`](https://github.com/taiki-e/cache-cargo-install-action)) that caches this instead of installing it via curl every time?

Reply:
That is fine enough for me. I lose track of all the different actions available. This is exactly what we do for the other wasmcloud interfaces though